apps/s_client.c

```diff
@@ -2153,22 +2143,12 @@ int s_client_main(int argc, char **argv)
         goto end;
     }
 #endif
-#ifndef OPENSSL_NO_QUIC
-    if (isquic && tfo) {
-        BIO_printf(bio_err, "%s: QUIC does not support the -tfo option\n", prog);
-        goto end;
-    }
-    if (isquic && alpn_in == NULL) {
-        BIO_printf(bio_err, "%s: QUIC requires ALPN to be specified (e.g. \"h3\" for HTTP/3) via the -alpn option\n", prog);
-        goto end;
-    }
-#endif
 
     if (tfo)
         BIO_printf(bio_c_out, "Connecting via TFO\n");
  re_start:
     if (init_client(&sock, host, port, bindhost, bindport, socket_family,
-                    socket_type, protocol, tfo, !isquic, &peer_addr) == 0) {
+                    socket_type, protocol, tfo, 1, &peer_addr) == 0) {
```

Comment:
This is the only one caller, so the `doconn` parameter can be dropped. This simplifies the function slightly.